From 40102b33e1c4ab80fb710f7b91595cce6a0943a0 Mon Sep 17 00:00:00 2001
From: Jannick <thirdedition@gmx.net>
Date: Wed, 25 Nov 2020 17:55:01 +0100
Subject: [PATCH 2/2] igraph-0.8.4: define DECLDIR to void string for GNUC
 compilers.

NB: DECLDIR appears to be required by MSVC, so this #define block could
cover #ifdef _MSC_VER only.
---
 include/igraph_decls.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/igraph_decls.h b/include/igraph_decls.h
index 38108bc22..af5d36d35 100644
--- a/include/igraph_decls.h
+++ b/include/igraph_decls.h
@@ -13,7 +13,7 @@
  * Reference: https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros */
 #undef DECLDIR
 #if defined (_WIN32)
-    #if defined (__CYGWIN__)
+    #if defined (__GNUC__)
         #define DECLDIR /**/
     #else
         #ifdef IGRAPH_EXPORTS
-- 
2.29.2.windows.2

